vtex/billing-app
# Make metrics URL for Billing

#### What is the purpose of this pull request?
Export to Billing the full URL for passively accumulated metrics, e.g. spending on google ads by `vtex.google-shopping`

#### What problem is this solving?
Billing does not know *a priori* how public IO URLs are formed. This addition solves this by dinamically writing them to the policy sent.

#### How should this be manually tested?
1. Clone/Pull this repo/branch and link this app in your workspace
```shell
$ vtex link --verbose
```
2. Purchase `vtex.google-shopping` by doing
```shell
$ vtex install vtex.google-shopping -r vtex --verbose
```
3. Check your contract at `{{account}}.myvtex.com/admin/billing/contracts`. It should have the full URL `http://{{account}}.myvtex.com/_v/billing` instead of `/_v/billing`

#### Types of changes
